corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -140,7 +140,7 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
         <td class="module-case-list-column"><img class="module-icon" src="<%- resolveUri(datum) %>"/></td>
       <% } else if (styles[index].displayFormat === 'ClickableIcon') { %>
         <td class="module-case-list-column">
-          <img class="module-icon clickable-icon" src="<%- resolveUri(datum) %>"/>
+          <img alt="<%- headers[index] %>" class="module-icon clickable-icon" role="button" src="<%- resolveUri(datum) %>" tabindex="0"/>
```

Comment:
I don't believe the `tabindex="0"` is necessary here _for accessibility._ Case list/tiles are already navigable with control+option+arrow keys (on Mac, possibly different on other OS). However, navigating to icons via "tab" was specifically called out in the ticket, and that's what this property does.